packages/web-react/src/types/shared/rest.ts

```diff
@@ -0,0 +1,5 @@
+type restValueType = unknown;
+
+export interface RestProps {
+  [k: string]: restValueType;
```

Comment:
Is there any problem not using `unknown` here instead of defining another type? I also suggest using PascalCase in the naming types. Now it looks like a variable.

```suggestion
  [key: string]: TransferringPropsValue;
```

Also, please use a full variable name instead of just `k` for better readability and understanding of the type. Thanks.